docs/user-guide/how-to.md

```diff
@@ -366,7 +366,15 @@ can use any of the following annotations:
 
 * **`propagate.hnc.x-k8s.io/none`**: Setting `none` to `true` (case insensitive)
   will result in the object not propagating to _any_ descendant namespace. Any
-  other value will be rejected.
+  other value will be rejected. The behavior would be the same as setting
+  [synchronization mode](#modify-the-resources-propagated-by-hnc) to 
+  `AllowPropagate` with no selectors at all.
+
+* **`propagate.hnc.x-k8s.io/all`**: Setting `all` to `true` (case insensitive)
+  will result in the object propagating to _any_ descendant namespace. Any
+  other value will be rejected. The behavior would be the same as setting
+  [synchronization mode](#modify-the-resources-propagated-by-hnc) to 
+  `Propagate` with no selectors at all.
```

Comment:
I think we should add a new paragraph below this list saying something like: "Warning: only use one of the propagation annotations on any one object at a time. Interactions between multiple propagation annotations is undefined and may change from release to release."

Reply:
Interactions between annotations are checked in the [integ testing ](https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/internal/objects/reconciler_test.go#L42-L176)

You think the behavior might change in the future so a warning would be better?

Reply:
Oh, I forgot about that. If it's straightforward to document, then maybe you can document it? Otherwise we can tell people not to rely on it :)

Reply:
I've added a warning for now, might doc it in a future PR though